### ISSUE TITLE
Feature request: reordering tabs

### ISSUE THREAD

**Issue by mediochrea** [octarinedev/cluster#3]
It would be extremely useful to be able to reorder tabs in Cluster, which would reorder them in the actual window as well. I'm not too well-versed in Chrome's API, so I'm not sure if the latter is doable or not, but it would be pretty awesome if it is.

**Comment by mikhoul** [comment]
I agree with you in meantime for reorder/close/search easily your tabs in the current windows you can use my extension here: https://github.com/mikhoul/TabsPaneMikhoulFork .

It's optimized for users that use lot of tabs like me. :octocat:

**Comment by octarinedev** [comment]
This has been on the list for the awhile and with sync being added there is time to start checking off some features.
